sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,5 @@
+select "countries"."name",
+      count("cityId")
+  from "countries"
+  join "cities" using ("countryId")
+  group by "countries"."name"
```

Comment:
You could also group by countryId, which would work properly in the unlikely case that two countries have the same name.